ibmdbpy4nps/ae/client code examples/stock_prediction_modelscoring_wlm_nps.py

```diff
@@ -4,15 +4,22 @@
 from ibmdbpy4nps import IdaDataBase, IdaDataFrame
 from ibmdbpy4nps.ae import NZFunGroupedApply
 
-dsn = 'bank'
+#dsn = 'bank'
+dsn='olaf'
```

Comment:
The name of the data source should be neutral, and not related to a person. So I would keep the name "bank" for the dsn.